README.md

```diff
@@ -36,11 +36,11 @@ ESPEasy_mega-20190225_normal_ESP8285_1M.bin  | ESP8285 with 1MB flash  | Stable
 ESPEasy_mega-20190225_test_ESP8285_1M.bin    | ESP8285 with 1MB flash  | Stable + Test               |
 ESPEasy_mega-20190225_dev_ESP8285_1M.bin     | ESP8285 with 1MB flash  | Stable + Test + Development |
 
-## More info
+## Documentation & more info
 
-Details and discussion are on the "Experimental" section of the forum: https://www.letscontrolit.com/forum/viewforum.php?f=18
+Our new, in-depth documentation can be found at [ESPEasy.readthedocs.io](https://espeasy.readthedocs.io/en/latest/). Automatically built, so always up-to-date according to the contributed contents. The old Wiki documention can be found at [letscontrolit.com/wiki](https://www.letscontrolit.com/wiki/index.php?title=ESPEasy).
 
-Automated builds of the (new) documentation can be found at [ESPEasy.readthedocs.io](https://espeasy.readthedocs.io/en/latest/)
+Additional details and discussion are on the "Experimental" section of the forum: https://www.letscontrolit.com/forum/viewforum.php?f=18
```

Comment:
Not sure if the "Experimental" part should be there.

Reply:
Well, the whole „Experimental“ thing isn't _my_ addition. That’s what I took over from the existing live version:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/3104762/92741566-0ba54c80-f37f-11ea-8941-de92b81b648c.png">

Just modify that line, if it isn't accurate.

Reply:
Note that I've swapped the order of those two phrases, as the actual documentation over at readthedocs.io is the main thing here, and discussion and further stuff are additional extras, not vice-versa.